scripts/skills/injury/rf_heat_stroke_injury.nut

```diff
@@ -0,0 +1,51 @@
+this.rf_heat_stroke_injury <- this.inherit("scripts/skills/injury/injury", {
+	m = {
+		FatigueCostMult = 1.25,
```

Comment:
Should be `FatigueEffectMult` because it applies to all fatigue built, not just from skills with fatigue cost.

Reply:
FatigueCostMult does not mention `skills`. I find it a very neutral term and always see it as including cost for moving around in it.
Using skills and moving around costs fatigue.

Other sources of fatigue build-up like getting hit, encumbrance or special triggered costs from skills are not influenced (en garde, rebuke)

Reply:
Same [comment as on the other conversation](https://github.com/Battle-Modders/mod-reforged/pull/292#discussion_r1566774650). If you are changing the `FatigueEffectMult` then this variable should be named similarly to make that clear.